WeatherPlannerDemo/Localization/LocalizationService.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+enum LocalizationKey: String {
+    case helloMessage
+    
+    var string: String {
+        return rawValue.localized()
+    }
+}
+
+extension LocalizationKey {
+    static func getLocalizationKey(for string: String) -> LocalizationKey? {
+        return LocalizationKey(rawValue: string)
+    }
+}
+
+extension String {
```

Comment:
Move this extension into separate file.

Reply:
Have this in the feature/city_details PR